modules/react/select/lib/Select.tsx

```diff
@@ -69,17 +79,16 @@ export const SelectInput = createSubcomponent(TextInput)({
     model
   ) => {
     return (
-      <InputGroup width={width} data-width="ck-formfield-width">
+      <InputGroup data-width="ck-formfield-width">
         {inputStartIcon && model.state.selectedIds.length > 0 && (
-          <InputGroup.InnerStart pointerEvents="none">
+          <InputGroup.InnerStart {...selectIconsStencil()}>
```

Comment:
Does this need `mergeStyles`? Or are we good with spreading this here?

Reply:
Not here because the consumer will never have access to this element. I call mergeStyles where the consumer can pass elemProps too.